app/src/views/private/components/image-editor.vue

```diff
@@ -147,6 +149,19 @@ function useImage() {
 	function save() {
 		saving.value = true;
 
+		const formData = new FormData();
+
+		if (localDragMode.value === 'focal_point') {
+			const data = cropperInstance.value?.getData();
+			const x = (data?.x ?? 0) + (data?.width ?? 0) / 2;
+			const y = (data?.y ?? 0) + (data?.height ?? 0) / 2;
+			// Note: Required to include other fields before the `file` key
+			formData.append('focal_point', JSON.stringify({ x, y } satisfies File['focal_point']));
```

Comment:
Should we 'just' do this as a regular JSON request? Right now we're technically re-uploading the whole file just to set the `focal_point` database column

Reply:
Yes we did it like that in the past too and I preserved this behaviour. I assume we did that so that we apply transformations like rotations and mirroring without needing to explicitly check for them(?). I guess it'd be also nice then to disable the saving button if there are no actual changes!

Reply:
Rotation/mirror/crop etc all modify the original image, which is then replaced on disk. This is just metadata in the db though so that should go elsewhere 🙂

Reply:
Update: 

1. Save button now correctly gets disabled if there are no changes
2. Refactored persistence to collapse to one request if both the image and focal point change, but stay separate otherwise
3. Leave Image editor open if there are errors instead of closing